Sources/ProjectSpec/TargetSource.swift

```diff
@@ -4,8 +4,13 @@ import PathKit
 
 public struct TargetSource: Equatable {
     public static let optionalDefault = false
-
-    public var path: String
+    
+    public var path: String {
+        didSet {
+            path = (path as NSString).standardizingPath
```

Comment:
As far as I remember didSet is not called from the initializer, so you'd need to do this transformation in the `public init(path: String ...)` and `init(jsonDictionary: JSONDictionary)`

Reply:
Ah interesting. I'm sure you saw my comment above about the last time I wrote Swift code. Interestingly, I tested this way and I couldn't repeat it. I wonder if something else is setting this property and that's what's triggering the bug? 

Either way, I'll add to the constructors.